src/RequestsLibrary/RequestsKeywords.py

```diff
@@ -201,16 +294,20 @@ def get_request(self, alias, uri, headers=None, params={}, allow_redirects=None,
         `headers` a dictionary of headers to use with the request
         
         `timeout` connection timeout
-        """
+        """        
         session = self._cache.switch(alias)
         params = self._utf8_urlencode(params)
         redir = True if allow_redirects is None else allow_redirects
-        response = self._get_request(session, uri, headers, params, redir, timeout)
+        try:            
+            response = self._get_request(session, uri, headers, params, redir, timeout)
+        except:                        
+            raise Exception("host=" + self.host + " uri: "+ uri + " not responding")
```

Comment:
Yep the RequestException is testing all the required ambiguous exception that occurred while handling http request

``` Python
class RetryException(RequestException):
    pass
```